### DISCUSSION TITLE
Lots of warnings > Seeking plots took too long: 34.44073 seconds!

### DISCUSSION THREAD

**Discussion by jkoenig72** [Chia-Network/chia-blockchain#6661] [Support]
Hi,

Just need some hints on how to improve my setup - I'm using chiadog (https://github.com/martomi/chiadog)

- average: 1.12s over 2999 searches
- over 5s: 63 occasions (2.1%)
- over 15s: 43 occasions (1.4%)
Plots 🌱: 1781, new: 16
Eligible plots 🥇: 3.47 average
Skipped SPs ⚠️: 1 (0.03%)

I get lots of those alerts >5 sec, normally in the range of 20 - 30 sec., how can I see which plot is responsible for that? How bad is this?

Also - those numbers above, are they in general ok? Have not got any chia so far, just wondering if this is bad luck or the setup is weak.

Any feedback?

Regards
Joerg

**Comment by loppefaaret** [comment]
you want to get those numbers down!
each plot with a scan time over 5 seconds, are being noted in `debug.log` on an ERROR level entry from the harvester

**Comment by BrandtH22** [comment]
@jkoenig72 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
